README.md

```diff
@@ -16,6 +16,40 @@
 pip install nebari-plugin-self-registration
 ```
 
+## User Registration
+How to register for the playground:
+
+- Navigate to your nebari domain.
+<p align="center">
+  <img src="images/main-landing.png" />
+</p>
+
+- Click the link at "To register for an account, click here." or go to https://{your-domain-name}/registration
+
+<p align="center">
+  <img src="images/account-register.png" />
+</p>
+
+- Enter your email address and coupon code.
+
+- After clicking "Submit" follow the instructions to login with your temporary password. By clicking the "Login" button, it will take you to a Welcome page where you can sign in with Keycloak.
+
+<p align="center">
+  <img src="images/account-register.png" />
+</p>
+
+- After you have entered a new password, you will receive an email from "donotreply@jatic.net".  Follow the verification instructions and you will be taken to the playground.
```

Comment:
```suggestion
- After you have entered a new password, you will receive a verification email.  
```

Reply:
Sorry, we missed that email address.  Good catch.